docs/source/documentation/platform/infrastructure/run-data-product-in-platform.html.md.erb

```diff
@@ -54,28 +71,63 @@ The following are the assumptions and risks. This will be mitigated eventually.
 
 3. Access control to various tools and users are out of scope for this work.
 
-### Notes
 
-##### Bucket
+### Scope and Limitations
+
+With this being used as a PoC it has quite a narrow scope and the following limitations:
+
+* Raw data are a full-load snapshot at every upload.
+* Raw data are csv format.
+* Data producers cannot include transformations of the data.
+* An extraction_timestamp - used as the partition - is the only column added to the data during the curation step (conversion to parquet). There is no SCD2.
+* No schema changes can be made once a data product is created, users must create a new product
 
-1. Users will be unaware that there is a bucket (Data Platform bucket), this is internal to the Data Platform.
+### Athena vs. Glue
+Athena is cheaper than using Glue for compute as you are charged for amount of data scanned per query ($5 per TB).
+Cost of lambda execution is also relatively inexpensive at $0.0000083 per second with 512 Mb memory.
 
-2. There is only one bucket (Data Platform bucket). Everything else will be sub folders of the same bucket
+Initial testing has shown Athena as compute to be a lot quicker than using a Glue Job for moderately sized data.
+Further benchmarking is required as scale and complexity increases.
+
+### Additional developments/ improvements
+
```

Comment:
Who is the audience for this section? Is it for the team or wider stakeholders? I'm just wondering this duplicating info that should live in the epics/roadmap?

Reply:
Removed - now this only resides in the internal version of this document (on confluence)